src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -51,53 +56,104 @@ public boolean isMaintainer() {
     return this.isMaintainer;
   }
 
-  //TODO(): get correct moderator status for organization
+  public String getUserEmail() {
+    return this.email;
+  }
+
+  public boolean isLoggedIn() {
+    return this.isLoggedIn;
+  }
+
+  // TODO(): get correct moderator status for organization
   public boolean isModerator(long organizationId) {
       return false;
   }
 
-  public static GivrUser getUserByIdFromDatastore(String userId) {
+  public static boolean checkIfUserWithPropertyExists(String propertyName, String propertyValue) {
```

Comment:
recommend removing this fiunction and just using

getUserFromDataStore(...) == null